tests/unit/test_utils.py

```diff
@@ -102,12 +104,18 @@ def test_already_snake_cased(self):
 
     def test_special_cases(self):
         # Some patterns don't actually match the rules we expect.
-        self.assertEqual(xform_name('SwapEnvironmentCNAMEs'), 'swap_environment_cnames')
-        self.assertEqual(xform_name('SwapEnvironmentCNAMEs', '-'), 'swap-environment-cnames')
-        self.assertEqual(xform_name('CreateCachediSCSIVolume', '-'), 'create-cached-iscsi-volume')
-        self.assertEqual(xform_name('DescribeCachediSCSIVolumes', '-'), 'describe-cached-iscsi-volumes')
-        self.assertEqual(xform_name('DescribeStorediSCSIVolumes', '-'), 'describe-stored-iscsi-volumes')
-        self.assertEqual(xform_name('CreateStorediSCSIVolume', '-'), 'create-stored-iscsi-volume')
+        self.assertEqual(xform_name('SwapEnvironmentCNAMEs'),
+                         'swap_environment_cnames')
+        self.assertEqual(xform_name('SwapEnvironmentCNAMEs', '-'),
+                         'swap-environment-cnames')
+        self.assertEqual(xform_name('CreateCachediSCSIVolume', '-'),
+                         'create-cached-iscsi-volume')
+        self.assertEqual(xform_name('DescribeCachediSCSIVolumes', '-'),
```

Comment:
In the future, please at least have separate commits for functional changes vs. code cleanup.

It doesn't have to be an entirely separate PR (though that's helpful), but it clutters PRs when reviewers have to sift through what's relevant to your changes vs. what's not.  I know I do this sometimes to so feel free to remind me if you see me doing it.